py/PulumiPlugin.yaml

```diff
@@ -1 +1,4 @@
-runtime: python
+runtime:
+  name: python
+  options:
+    virtualenv: venv
```

Comment:
Does this have any effect?

Reply:
Let's discuss that in this other PR: https://github.com/mikhailshilkov/comp-as-comp/pull/5